java/find-closest-facility-to-an-incident-interactive/README.md

```diff
@@ -2,27 +2,29 @@
 
 Find a route to the closest facility from a location.
 
-![Find closest facility to an incident interactive App](find-closest-facility-to-an-incident-interactive.png)
+![Image of find closest facility to an incident interactive](find-closest-facility-to-an-incident-interactive.png)
 
 ## Use case
 
 Quickly and accurately determining the most efficient route between a location and a facility is a frequently encountered task. For example, a paramedic may need to know which hospital in the vicinity offers the possibility of getting an ambulance patient critical medical care in the shortest amount of time. Solving for the closest hospital to the ambulance's location using an impedance of "travel time" would provide this information.
 
 ## How to use the sample
 
-Click near any of the hospitals and a route will be displayed from that clicked location to the nearest hospital.
+Tap near any of the hospitals and a route will be displayed from that tapped location to the nearest hospital.
 
 ## How it works
```

Comment:
let's just ensure all these steps have periods at the end of them, some are missing from the design